plugins/clipboard/plugin.js

```diff
@@ -568,6 +568,24 @@
 						paste: stateFromNamedCommand( 'paste' )
 					};
 				} );
+
+				// Adds 'touchend' integration with context menu paste item (#1347).
+				var pasteListener = null;
+				editor.on( 'menuShow', function( evt ) {
+					// Remove previous listener.
+					if ( pasteListener ) {
+						pasteListener.removeListener();
+						pasteListener = null;
+					}
+
+					// Attach new 'touchend' listeners to context menu paste items.
+					var item = evt.data.menu.findItemByCommandName( 'paste' );
+					if ( item && item.element ) {
+						pasteListener = item.element.on( 'touchend', function() {
+							editor._.forcePasteDialog = true;
```

Comment:
`pasteListener` is attached on `menuShow`, so every time the menu is open, new listener is attached (menu is rerendered every time so it makes sense). Removing `pasteListener` on `touchend` event will only work if it will be fired (someone "touches" the menu item), but there are man different ways of closing context menu without `touchend` and then listener will not be removed properly.